### ISSUE TITLE
Integrated Thebe Status & Launch Buttons

### ISSUE THREAD

**Issue by stevejpurves** [executablebooks/thebe#470]
The Thebe repository provides examples in both `rst` (via it's docs) and `html`.

Within those examples, code is provided that allow users of the library to add their own status buttons and launch buttons. 

Separately the `sphinx-thebe` library implements a  status and launch button on top of dynamically loading `thebe`.

In a draft PR https://github.com/executablebooks/thebe/pull/459 I have taken a version of the kernel status widget from the examples and pulled it into the library, so it can be added to a page by adding a `div` tag with the expected class and supplying the config option.

We could do the same with a launch/activate button, meaning downstream users would not have to implement this.

questions:

 - Is there a case for adding a launch button?
 - Should we actually pull in the launch button from `sphinx-thebe`, and modify the latter to use that built in version?

**Comment by choldgraf** [comment]
I think this is a cool idea - how would this work at the implementation level? Would the bootstrap just look for a few special DOM elements and, for the ones it find, process them to have the right hooks and such, and add some CSS?

**Comment by moorepants** [comment]
Having the status indicators be a default part of the interface is preferable. We had a fair amount of requests from users at Libretexts for more information about what was going on after "run" was pressed. Given that new images can take so long to load, it's really essential. We have a status indicator in the lower left and now that the little spinner is there from Steve's work, the users seem to have enough feedback. You can see ours here:

https://query.libretexts.org/Sandboxes/jupyterteam_at_ucdavis.edu

**Comment by stevejpurves** [comment]
@moorepants yes I see it! that is nice and ys what it would be nice to provide out of the box.

@choldgraf yes, having DOM elements with special classes and thebe would "attach" new elements there, with minimal styling, giving the owner of the page ability to customize and style easily.
